development/contracts/ForkManager.sol

```diff
@@ -389,7 +389,7 @@ contract ForkManager is Arbitrator, IERC20, ERC20 {
     // This can be used to freeze operations pending the outcome of a governance question
     // TODO: An earlier bond should also be enough if you don't call this right away
     function _verifyMinimumBondPosted(bytes32 question_id, uint256 minimum_bond) 
-    internal {
+    internal view {
```

Comment:
If you're doing a bunch of requires is that "view"?

Reply:
yes, view means that its only reading, and no writing. Depending on what is happing in the "requires", it can be view or storage operations. Here its view only